Unity/Assets/GameSystems/Unity/Components/Spawner/Spawner.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Spawner : MonoBehaviour
+{
+    [SerializeField] GameObject prefab;
```

Comment:
Need a private after the [serializeField], as mentioned here - https://github.com/AdolfoPSakata/UnityTraining-2/pull/1